mix.exs

```diff
@@ -54,9 +66,11 @@ defmodule Nimble.MixProject do
   # See the documentation for `Mix` for more info on aliases.
   defp aliases do
     [
+      lint: ["format --check-formatted", "credo"],
       "ecto.setup": ["ecto.create", "ecto.migrate", "run priv/repo/seeds.exs"],
       "ecto.reset": ["ecto.drop", "ecto.setup"],
-      test: ["ecto.create --quiet", "ecto.migrate", "test"]
+      test: ["ecto.create --quiet", "ecto.migrate", "test"],
+      coverage: ["sobelow --config", "coveralls.html --raise"]
```

Comment:
Is the output here is `html` output @Tolsee 

![image](https://user-images.githubusercontent.com/11751745/87284073-2d6d8900-c520-11ea-8534-023da6f2f48f.png)

I'm trying with this setup and I got a different result

![image](https://user-images.githubusercontent.com/11751745/87284244-5726b000-c520-11ea-9a2e-9471b1b0a3e8.png)

Reply:
This is a summary shown in command line, HTML output will be add in `cover/excoveralls.html`